package.json

```diff
@@ -24,7 +29,7 @@
     "webpack-dev-server": "^4.11.1"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
```

Comment:
- [ ] [OPTIONAL] you can Add a `--coverage` flag to your test script in package.json. That way you will see the percentage coverage of your application code every time you run a test:

```JS
    ...
    "scripts": {
        "test": "jest --coverage",
        ...
    }
```
When writing tests the closer you get to 100% coverage the better.